esi_leap/resource_objects/ironic_node.py

```diff
@@ -15,12 +15,24 @@
 from esi_leap.common import ironic
 import esi_leap.conf
 from esi_leap.resource_objects import base
+from ironicclient.common.apiclient import exceptions
 
 
 CONF = esi_leap.conf.CONF
 _cached_ironic_client = None
 
 
+class FakeIronicNode(object):
+    def __init__(self):
+        self.name = 'fake-node'
+        self.owner = 'fake-project-uuid'
+        self.uuid = 'fake-uuid'
+        self.properties = {'lease_uuid': 'fake-lease-uuid'}
+        self.lessee = 'fake-project-uuid'
+        self.maintenance = False
+        self.provision_state = 'active'
```

Comment:
These values should not have 'fake' in them; perhaps 'unknown' or 'n/a'

Reply:
Okay!